web/client/components/data/identify/Identify.jsx

```diff
@@ -118,9 +120,37 @@ class Identify extends React.Component {
         closeGlyph: "1-close",
         className: "square-button",
         allowMultiselection: false,
-        currentLocale: 'en-US'
+        currentLocale: 'en-US',
+        fullscreen: false
```

Comment:
The fullscreen property seems to disable fullscreen touch when true. It is never used? Can you explain this?

Reply:
It used to disable fullscreen on desktop in localConfig.
It removes function doble click/touch from the below renderHeader method.